doc/developer-notes.md

```diff
@@ -124,6 +124,23 @@ can be very difficult to track down. Compiling with -DDEBUG_LOCKORDER (configure
 CXXFLAGS="-DDEBUG_LOCKORDER -g") inserts run-time checks to keep track of which locks
 are held, and adds warnings to the debug.log file if inconsistencies are detected.
 
+**Memory Profiling**
+
+Bitcoin Unlimited can be compiled with the libtcmalloc allocation library and
```

Comment:
The configuration seems to only enable it for Linux at the moment. Maybe mentioned "can be compiled on Linux with the ..." .

It seems as if the library is [available on other platforms](https://github.com/gperftools/gperftools) - I suppose it's been enabled only for Linux initially due to testing resource scarcity.